lld/test/ELF/aarch64-thunk-bti.s

```diff
@@ -0,0 +1,482 @@
+// REQUIRES: aarch64
+// RUN: rm -rf %t && split-file %s %t && cd %t
+// RUN: llvm-mc -filetype=obj -triple=aarch64 asm -o %t.o
```

Comment:
After `rm -rf %t && split-file %s %t && cd %t`, `%t` can be completely avoided.
For example, name an output file `a.o` and output as `out` instead `%t.exe`